frontend/src/concepts/pipelines/content/compareRuns/metricsSection/markdown/useFetchMarkdownMaps.ts

```diff
@@ -39,18 +39,27 @@ const useFetchMarkdownMaps = (
         .map(async (path) => {
           const { run } = path;
           let sizeBytes: number | undefined;
-
-          const url = await getStorageObjectUrl(path.linkedArtifact.artifact).catch(
-            () => undefined,
-          );
+          let url: string | undefined;
+          if (
+            path.linkedArtifact.artifact.getType() === ArtifactType.MODEL ||
+            path.linkedArtifact.artifact.getType() === ArtifactType.HTML
```

Comment:
why is HTML download and not preview

Reply:
or should markdown be here

Reply:
actually i think only markdown should be here and everything else we wont use. This output is used as a src in a iframe and that needs to be the download url, not preview.

Reply:
Yes, you are right. It's a mistake, so I will change the model type to the markdown type.